cas-server-support-duo/src/main/java/com/duosecurity/duoweb/Util.java

```diff
@@ -7,6 +7,15 @@
 import javax.crypto.spec.SecretKeySpec;
 
 public class Util {
+    /**
+     * TODO: 12/10/15   move to DuoWeb
```

Comment:
Thanks. Removing Duo is simply not an option. Even though they make it very difficult to use their API in a sane way, they are by far the most popular option, and we want to make sure we support them. Most deployments of MFA would either opt for Duo or RSA. Both of which we support, or should.